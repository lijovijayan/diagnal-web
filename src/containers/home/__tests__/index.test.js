
import { Home } from '../index';

describe('testing home module', () => {
    it('rendered properly', () => {
        const home = <Home />
        expect(home).toMatchSnapshot()
    })
})
