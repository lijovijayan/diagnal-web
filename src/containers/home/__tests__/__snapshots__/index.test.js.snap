// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing home module rendered properly 1`] = `
<Home
  alt=""
  className=""
  src=""
/>
`;
